Lamina/include/GL/window.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef LM_WINDOW
+#define LM_WINDOW
+
+#include "../Lamina/include/GL/colors.hpp"
+#include "../Lamina/include/Math/vertex.hpp"
+#include <glad/glad.h>
+#include <GLFW/include/GLFW/glfw3.h>
+#include <iostream>
+#include <string>
+
+//define lamina window types
+#define LM_WINDOW_RESIZABLE			0b00000001
+#define LM_WINDOW_VISIBLE			0b00000010
+#define LM_WINDOW_DECORATED			0b00000100
+#define LM_WINDOW_FOCUSED			0b00001000
+#define LM_WINDOW_MAXIMIZED			0b00010000
+#define LM_WINDOW_DOUBLE_BUFFER		0b00100000
+#define LM_WINDOW_FULLSCREEN		0b01000000
+#define LM_WINDOW_SCALE_TO_MONITOR	0b10000000
+#define LM_WINDOW_DEFAULT			0b00101111
+#define LM_WINDOW_NON_RESIZABLE		0b00101110
+
+
+namespace lm
+{
+	struct WindowSettings
+	{
+		int aliasingLevel = 0;
```

Comment:
I haven't read far enough to know what this does exactly, but generally "aliasing" is bad and is avoided through something called "antialiasing" which I'm assuming is what value controls. If that's not the case, you can ignore this comment. Otherwise, I'd suggest renaming this to `antialiasingLevel`.